src/main/java/com/rusticisoftware/tincan/Attachment.java

```diff
@@ -79,8 +86,23 @@ public Attachment(JsonNode jsonNode) throws URISyntaxException, MalformedURLExce
         if (! fileUrlNode.isMissingNode()) {
             this.setFileUrl(new URL(fileUrlNode.textValue()));
         }
+
+        JsonNode contentNode = jsonNode.path("content");
+        if (! contentNode.isMissingNode()) {
+            this.setContent(contentNode.binaryValue());
+        }
```

Comment:
I agree with you on this one. I think the separate constructor idea would make more sense than expecting it in a JSON node.